flight/PiOS/Common/pios_hal.c

```diff
@@ -230,14 +231,14 @@ void PIOS_HAL_Panic(uint32_t led_id, enum pios_hal_panic code) {
 			PIOS_LED_Toggle(led_id);
 			PIOS_DELAY_WaitmS(175);
 		}
-		PIOS_DELAY_WaitmS(200);
+		PIOS_DELAY_WaitmS(175);
```

Comment:
Ahhh, LSI clock (which IWDG uses) is 32 kHz on F4, and 40 kHz on F1/F3, PIOS_WDG assumes they're all 32 kHz. :/